src/react-native/02-setting-up-your-environment/setting-up-your-environment.md

```diff
@@ -2,47 +2,92 @@
+
+### Install Node.js and npm
+
+Our solution requires the [Node.js](https://nodejs.org/) JavaScript runtime environment. Node.js and its included package manager `npm` will be used to do a variety of tasks including: installing required packages, running the development server, executing tests, and building the application for deployment.
+
+### Install React Native and Dependencies
+
+There are several dependencies used to create, build, and deploy a React Native application, and the specifics vary based on your development OS (MacOS, Windows, or Linux) and target OS (Android or iOS). Building for iOS require a MacOS computer, so this training only focuses on building for Android, though most of the techniques apply to all targets. Expo Go is a very convenient way to build simple React Native apps, though we will be using the official React Native CLI for this course, due to Expo Go's limitations.
 
 ### Setup 1
 
-TODO
+This course requires Node.js version 20 (we always suggest using the long-term support releases).
+
+The simplest approach is to download the [Node.js installer](https://nodejs.org/en/download) for your operating system. Any installation of Node.js will also include npm.
+
+Alternatively, you can [install Node.js via package manager](https://nodejs.org/en/download/package-manager); this is the reccomendation for most professional setups.
+
+During the installation process you may be prompted to make selections or install prerequisites, use the default selections and proceed with the installation.
+
+The installation process can take 10 to 15 minutes to complete.
 
 ### Verify 1
 
-TODO
+✏️ Run the following command in a terminal to verify Node.js is installed correctly.
+
+🔦 _If you encounter an error running `node --version` you may need to restart for node to be completely installed._
+
+```shell
+node --version
+```
+
+The output of the command will be the current version of Node.js, we expect it to start with either "v18" or "v20".
+
+✏️ Run the following command in a terminal to verify `npm` is installed correctly:
+
+```shell
+npm --version
+```
+
+The output of the command will be the current version of `npm`, we expect it to start with either "9" or "10".
+
+### Setup 2
+
+This course also requires several React Native dependencies, depending on your operating system.
 
-### Exercise 1
+Follow the setup instructions for [React Native CLI Quickstart for your development OS and an Android target OS](https://reactnative.dev/docs/environment-setup?guide=native&platform=android), including setting up a virtual device.
 
-TODO
+The installation process can take 30 minutes or more to complete.
 
-### Solution 1
+### Verify 2
 
-If you’ve implemented the solution correctly, the tests will pass when you run `npm run test`!
+✏️ Start the virtual device you created in the setup.
+✏️ Run the following commands in a terminal to verify React Native is installed correctly.
```

Comment:
These two lines run together in the rendered version.

But I think that’s beside the point. We’ll create the app on the next page, so I would remove this.